### PR TITLE
[AIS-15] - Insert text page

### PR DESCRIPTION
**Pull request by olivernygren**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,346 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```
Comment (fannymartensson) [inline yarn.lock:2]: We should only use npm in this project. Therefore delete yarn.lock files.

Comment (olivernygren) [reply yarn.lock:2]: Now deleted

Comment (fannymartensson) [reply yarn.lock:2]: I think there is one yarn.lock file left to delete.

Comment (olivernygren) [reply yarn.lock]: In the root folder, yes... Now deleted xD

File: ai-summary-front/src/pages/insert-text/InsertTextPage.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { Button, Grid, TextField, Typography } from '@mui/material';
+
+import useStyles from './styles';
+import { Page, PageHeading } from '../../components';
+import { theme } from '../../theme';
+import data from '../../assets/data/data.json';
```
Comment (boomboxbacardi) [inline ai-summary-front/src/pages/insert-text/InsertTextPage.tsx:7]: `import data from '../../assets/data/data.json';`

This file includes localizable strings. Switch file name to something more descriptive

**Review by fannymartensson — APPROVED** [review]

File: ai-summary-front/src/pages/insert-text/InsertTextPage.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+export const InsertTextPage = () => {
+  const classes = useStyles();
+
+  const [insertedText, setInsertedText] = useState('');
+
+  const textMaxLength = 3000;
```
Comment (boomboxbacardi) [inline ai-summary-front/src/pages/insert-text/InsertTextPage.tsx:14]: `const textMaxLength = 3000;`

My suggestion is that you import this from a config-file like we have in React Native. So it gets easier for us to access and change the config dynamically (e.g. environment variables in future)

**Pull request merged**